examples/mixture/plot_gmm_selection.py

```diff
@@ -94,6 +94,7 @@
 
 plt.xticks(())
 plt.yticks(())
-plt.title('Selected GMM: full model, 2 components')
+plt.title('Selected GMM: {} model, {} components'.format(
+    best_gmm.covariance_type, best_gmm.n_components))
```

Comment:
I agree an f-string would be nicer otherwise LGTM.

```suggestion
plt.title(f'Selected GMM: {best_gmm.covariance_type} model, '
          f'{best_gmm.n_components} components')
```